ServiceBased/JupyterNotebooks/ImportExportWorkflowItems/Workflow Manager - Import Export Items Asynchronously.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Export and Import Workflow Manager Items Asynchronously #\n",
+    "\n",
+    "This sample will export an existing Workflow Manager Item, create a new workflow item, then import the configuration to the newly created item in the same GIS. All this work will be done asynchronously."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Make connections ###"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import arcgis\n",
+    "from arcgis.gis.workflowmanager import WorkflowManager, WorkflowManagerAdmin\n",
+    "import time\n",
+    "\n",
+    "gis = arcgis.gis.GIS(url='https://hostname/portal', profile='my_profile')\n",
+    "wm_admin = WorkflowManagerAdmin(gis)\n",
+    "print(\"Created connection to workflow manager\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Export existing item asynchronously ###"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Export Item Asynchronously\n",
+    "\n",
+    "item = gis.content.search('title:\"Python Sample\"')[0]\n",
+    "export_execution = wm_admin.export_item(item, run_async=True, download_location='C:\\\\Users\\\\exampleUser\\\\Desktop\\\\')\n",
+    "\n",
+    "# Example: Can do additional computation while execution is running and before prompting for result\n",
+    "wm = WorkflowManager(item)\n",
+    "diagrams = wm.diagrams\n",
+    "\n",
+    "# Now process the export_execution - result() stops execution until the asynchronous work is finished and returns the last message received.\n",
```

Comment:
Do you think we need to change this to `blocks execution until`? Not sure if this would also require updates in the blog?